### PR TITLE
made setBackgroundColor public so you can set color through @ColorInt…

### PR DESCRIPTION
**Pull request by qijaz221**
… and not just @ColorRes. Useful when you have a color picked from a color picker. Also renamed original setBackgroundColor to  setBackgroundResColor which still accepts @ColorRes.

### CODE REVIEW

**Comment by JoaquimLey** [comment]
Hey sorry for the delay, I was away.

I'll get back to you later today, thanks for the PR in advance.

**Comment by qijaz221** [comment]
No worries, take your time.

File: faboptions/src/main/java/com/joaquimley/faboptions/FabOptions.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ public void setBackgroundColor(@ColorRes int backgroundColor) {
     }
 
 
-    private void setBackgroundColor(Context context, @ColorInt int backgroundColor) {
+    public void setBackgroundColor(Context context, @ColorInt int backgroundColor) {
```
Comment (JoaquimLey) [inline faboptions/src/main/java/com/joaquimley/faboptions/FabOptions.java]: So if we're making this as a public API maybe we should have the method as an Overload?

We're using different parameters so it won't clash.

**Comment by JoaquimLey** [comment]
@qijaz221 are you following up or should I take over?

**Comment by JoaquimLey** [comment]
@qijaz221 hope you don't mind but I might have to reject this **PR**.

I've waited a while to get some feedback from you but without any success

**Comment by JoaquimLey** [comment]
Closing due to inactivity of the assignee.

**Pull request closed without merging**